src/site/xdoc/server/config-blob.xml

```diff
@@ -0,0 +1,109 @@
+<?xml version="1.0"?>
+<!--
+  Licensed to the Apache Software Foundation (ASF) under one
+  or more contributor license agreements.  See the NOTICE file
+  distributed with this work for additional information
+  regarding copyright ownership.  The ASF licenses this file
+  to you under the Apache License, Version 2.0 (the
+  "License"); you may not use this file except in compliance
+  with the License.  You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing,
+  software distributed under the License is distributed on an
+  "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+  KIND, either express or implied.  See the License for the
+  specific language governing permissions and limitations
+  under the License.    
+-->
+<document>
+
+ <properties>
+  <title>Apache James Server 3 - WebAdmin Configuration</title>
+ </properties>
+
+<body>
+
+  <section name="Blob Configuration">
+      <p>Consult <a href="https://github.com/apache/james-project/blob/master/dockerfiles/run/guice/cassandra-rabbitmq/destination/conf/blob.properties">blob.properties</a> in GIT to get some examples and hints.</p>
+
+      <subsection name="Blob Store Configuration">
+          <p>
+              <a href="https://github.com/apache/james-project/blob/master/server/blob/blob-api/src/main/java/org/apache/james/blob/api/BlobStore.java">BlobStore</a>
+              is a component used to storing blob data, it can take InputStream or byte array as the input data to store, then return the corresponding
+              <a href="https://github.com/apache/james-project/blob/master/server/blob/blob-api/src/main/java/org/apache/james/blob/api/BlobId.java">BlobId</a>.
+              Clients can use BlobStore to retrieve stored data by the BlobId.
+              <dl>
+                  <dt><strong>implementation</strong></dt>
+                  <dd>
+                      Define the BlobStore implementation will be used in the product. Available implementations: cassandra, objectstorage, union.
+                      cassandra: blobs will be stored and retrieved at cassandra database.
+                      objectstorage: blobs will be stored and retrieved at open-stack swift.
+                      union: blobs will be stored and retrieved at the current BlobStore(open-stack swift), if fails or not found, it fallback to use the legacy BlobStore (cassandra)
```

Comment:
union is always working with these two implementations in the same order or is it configurable?
Why do you talk about "current", what does it mean?

Reply:
it's not configurable. union blobstore hard coded swift as the primary and cassandra as the secondary. 
I will replace "current" & "legacy" by "primary" & "secondary", looks easy to understand